requirements/torch_common.txt

```diff
@@ -2,4 +2,4 @@ cython==0.29.34
 wheel==0.40.0
 pillow==9.3.0
 numpy==1.24.3; sys_platform != 'win32'
-numpy==1.19.3; sys_platform == 'win32' #see https://tinyurl.com/y3dm3h86
+numpy==1.22.0; sys_platform == 'win32' #see https://tinyurl.com/y3dm3h86
```

Comment:
Please check if we can use >= instead and not have to pin the version

Reply:
Retaining fix recommended by dependabot since we currently don't test windows in CI.